onyo/lib/inventory.py

```diff
@@ -547,3 +551,10 @@ def get_faux_serials(self,
                 faux_serials.add(f'faux{serial}')
 
         return faux_serials
+
+    def _required_key_empty(self, asset: dict) -> bool:
+        """Whether `asset` has an empty value for a required key
```

Comment:
I would add the dot to the end of the line here (and probably add a linebreak in the definition), though this is of course not a blocker :)

Reply:
Done